### DISCUSSION TITLE
自定义的custom.yaml包括哪些内容

### DISCUSSION THREAD

**Discussion by jaesony** [kkkgo/PaoPaoGateWay#75] [问和答]
自定义的custom.yaml包括哪些内容，除了节点信息，规则需要吗？
客户端需不需要指定网关为paopaogateway？

**Comment by kkkgo** [comment]
1、取决于你需不需要规则。
2、如果你想全局代理就这么干。
3、建议先看[详细说明《FakeIP网关的工作原理》](https://blog.03k.org/post/paopaogateway.html)
